README.md

```diff
@@ -91,32 +105,26 @@ each installation:
    for API, Ingress, and DNS access.
 1. The BMC IPMI addresses and credentials for the 3 control plane
    machines.
-1. Optionally, a Network Time Protocol (NTP) server where the default
-   public server is not accessible
+1. If detected that the provisioning host is not sync with a time source, configure the 25G switch as a source via the DHCP service  on the `Storage` network. An optional script to set a source for the switch, will be provided. 
 
 ## Provisioning Host Setup
 
 The provisioning host must be a RHEL-8 machine.
 
-Make a copy of `config_example.sh` and set the required variables in
-there.
+### For a host not installed using the ISO:
+In the OpenShift subdirectory, create a copy of `config_example.sh` using the existing
+user as part of the file name. For example, `config_<username>.sh`. Once the file has
+been created, set the required PULL_SECRET variable within the shell script
 
-To install some required packages and the `oc` client:
+To install some required packages, configure `libvirt`, `provisioning` and `baremetal` bridges, from the top directory:
 
 ```sh
-make requirements
+make prep
 ```
 
-Note:
-
-1. This ensures that a recent 4.2 build of `oc` is installed. The
-   minimum required version is hardcoded in the script.
-
-To configure libvirt, and prepare the `provisioning` and `baremetal`
-bridges:
-
+### For all nodes, Create the cluster 
 ```sh
-make configure
+make OpenShift
 ```
 
```

Comment:
Can you add in the README file a note about using a LOGLEVEL environment variable to increase the loglevel output of the `openshift-install` command? Such as:

Note: In order to increase the log level ouput of `openshift-install`, a `LOGLEVEL` environment variable can be used as:
```
export LOGLEVEL="debug"
make OpenShift
```

This will fix Russel's comment in #51 

I can do a follow up PR if you don't want to add it into this big PR tho :)

Thanks!

Reply:
I'll attempt to get it in prior to @markmc review else may ask you to do the follow up